### PR TITLE
fix: featbit prebuilt environment variables

### PR DESCRIPTION
**Pull request by deleteLater**

### CODE REVIEW

**Review by pan93412 — APPROVED** [review]

File: prebuilts/featbit-data-analytics-server.toml [hunk 1]
```diff
@@ -14,9 +14,6 @@ port = 80
 type = "HTTP"
 
 [env]
-REDIS_HOST = { default = "${REDIS_HOST}" }
-REDIS_PASSWORD = { default = "${REDIS_PASSWORD}" }
-REDIS_PORT = { default = "${REDIS_PORT}" }
 MONGO_URI = { default = "${MONGO_CONNECTION_STRING}" }
 MONGO_INITDB_DATABASE = { default = "featbit" }
 MONGO_HOST = { default = "mongodb.zeabur.internal" }
```
Comment (pan93412) [inline prebuilts/featbit-data-analytics-server.toml:19]: I afraid if this is also unnecessary.

Comment (deleteLater) [reply prebuilts/featbit-data-analytics-server.toml:19]: Yes, it's weird, but the `MONGO_HOST` is used to perform liveness check. We can remove it if we set `CHECK_DB_LIVNESS` to `false`.

https://github.com/featbit/featbit/tree/main/modules/data-analytics#mongodb-settings

image

Comment (pan93412) [reply prebuilts/featbit-data-analytics-server.toml:19]: https://github.com/zeabur/zeabur/blob/main/prebuilts/mongodb.toml

Basically MONGO_HOST has been implicitly injected into this service. I do not think this is necessary to override this variable again.

Comment (deleteLater) [reply prebuilts/featbit-data-analytics-server.toml:19]: Aha, so we can safely remove the `MONGO_HOST` and `MONGO_URI` variables?

Comment (pan93412) [reply prebuilts/featbit-data-analytics-server.toml:19]: Yeah.

Comment (deleteLater) [reply prebuilts/featbit-data-analytics-server.toml:19]: resolved in https://github.com/zeabur/zeabur/pull/370

**Pull request merged**